source/Ship.cpp

```diff
@@ -1078,6 +1078,17 @@ int64_t Ship::ChassisCost() const
 
 
 
+// Get the full cost of this ship on the planet, depending on the outfit prices.
+int64_t Ship::LocalCost(const Planet *planet, const ConditionSet::Conditions conditions) const
```

Comment:
Why is `conditions` taken by value? Should be a reference, which will remove the need for the full include in the header.

Reply:
ConditionSet::Conditions is a class nested in a class so I have to include it

Reply:
It's a type alias for a `std::map` that I should have kept private to the class. Use that and you can remove the need for the include.